opensearch-operator/examples/opensearch-cluster.yaml

```diff
@@ -21,16 +21,10 @@ spec:
          cpu: "500m"
   confMgmt:
     smartScaler: true
-  security:
```

Comment:
Lets add, that if client is declaring `securityConfigSecret` == default, it will create that default secret.
when we just deleting it, it look like a waste of a good feature :)

Reply:
So @idanl21 I have modified the code in such a way that a user need not now mention `securityConfigSecret==default`, as if TLS is enabled by default operator will created a default securityConfigSecret secret, that has default values, so this way a user need not explicitly mention pre-created `securityConfigSecret name`. Once the TLS is added and `securityConfigSecret name` is not added, the operator wont fail to start the batch job, as it will create a default secret and uses it to run securityadmin.sh for security plugin. If user added `securityConfigSecret name`, then operator proceeded to use this secret and does not create a default one, this makes user inputs easy as need not add `securityConfigSecret == default`.